src/sqs/index.ts

```diff
@@ -40,24 +51,60 @@ export type QueueSettings = {
 
 	readonly longPollingIntervalSec?: number,
 
-	//TODO Add batching
+	/**
+	 * Timeout for processing a single message in seconds
+	 */
+	readonly timeoutSec: number;
```

Comment:
Javascript works in milliseconds and it's better to keep it the standard or else you risk the chance of setting something for 2 milliseconds.

Reply:
SQS Client works in seconds, plus there are other parameters which are set in seconds. Will keep seconds for consistency.